Source/common/SNTBlockMessage.m

```diff
@@ -123,6 +123,11 @@ + (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event {
       [formatStr stringByReplacingOccurrencesOfString:@"%file_sha%"
                                            withString:event.fileBundleHash ?: event.fileSHA256];
```

Comment:
Based on the description, should `%file_sha%` be changed to:
```suggestion
                                           withString:event.fileSHA256];
```

Reply:
Indeed, I thought I had done that already, thanks